INSTALL.md

```diff
@@ -52,22 +69,51 @@ To skip the secrets taks, just pass:
 
 The secret keys that have to be defined are:
 
-* backend_url
-* base_url
-* backend_token
-* secret_key
-* info_email
-* ssl_stapling_resolver
-* google_analytics
-
+* backend_url: The URL for requesting the backend. Ex: http://kernelci-backend/
+* base_url: The URL for requesting the frontend. Ex: http://kernelci-frontend/
+* backend_token: A GET+POST token. (See below for generating them)
+* secret_key:  should be set to a random string, it is used internally by Flask.
+* master_key: The password used for generating tokens before any admin token was created.
+* info_email:
+* ssl_stapling_resolver:
+* google_analytics:
 
-The `secret_key` value should be set to a random string, it is used internally
-by Flask.
 
 Other non-secrets variable might need to be defined, please look at the `templates` directories.
 
 ### Run Ansible
 ```
 $ cd <kernelci-backend-diretory>
 $ ansible-playbook -i hosts site.yml -e "@../secrets.yml" -l <$TARGET_NAME> 
-```
\ No newline at end of file
+```
+
+This will deploy the kernel-ci backend code into `/srv/$hostname`,
+intall all dependencies and set up an nginx host called `$hostname`.
+
+By default an S3-backup shell script and firewall rules via `ufw` will be
+installed as well. Skip them with:
+
+  --skip-tags=backup,firewall
+
+### Requirements
+
+Non exhaustive list of requirements is in the 'requirements.txt' file: those
+need to be installed via pip.
+For production, requirements.txt is sufficient. For development purpose, requirements-dev.txt
+will add extra package for testing.
+
+### Manual tasks
+ * You need to generate an admin token, this token will be used to generate others token (referenced as ADMINTOKEN below)
+```
+	curl -XPOST -H "Content-Type: application/json" -H "Authorization: SECRET_KEY" "localhost:8888/token" -d '{"email": "xxx@xxx", "username": "admin", "admin": 1}'
+```
+ * You need to generate a GET+POST token for the frontend. Set in /etc/linaro/kernelci-frontend.cfg BACKEND_TOKEN=xxxx
+```
+	 curl -XPOST -H "Content-Type: application/json" -H "Authorization: ADMINTOKEN" "localhost:8888/token" -d '{"email": "xxx@xxx", "username": "admin", "get": 1, "post": 1}'
+```
+
+ * You need to generate a LAB token for your LAVA lab. This token will be used along with the backend FQDN for CALLBACK Jobs.
+```
```

Comment:
Should probably add that this step is optional, and only needed when setting up a LAVA lab that interacts with the backend.